applications/views.py

```diff
@@ -174,3 +185,48 @@ def post(self, request, *args, **kwargs):
         application.save()
 
         return Response(status=status.HTTP_200_OK)
+
+
+class VideoInterviewsView(GenericAPIView):
+    """
+    Create an interview on Jobma, then redirect the user to it
+    """
+
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated, UserIsOwnerPermission)
+    lookup_field = "pk"
+    owner_field = "user"
+    queryset = BootcampApplication.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        """
+        Create the Interview, then POST to Jobma to create it there,
+        then return a URL for the user to go to
+        """
+        user = request.user
+        application = self.get_object()
+        job = get_object_or_404(Job, run=application.bootcamp_run)
+        interview, _ = Interview.objects.get_or_create(
+            applicant=user, job=job, defaults={}
+        )
+        interview_link = create_interview(interview)
+
+        # Assuming only one video submission step per bootcamp. TODO: is this a safe assumption?
```

Comment:
@noisecapella Couldn't we just add the `BootcampRunApplicationStep` id to the URL or to the body of the request? You could change the drawer action to add that id in the meta and pass it down to `TakeVideoInterviewDisplay`.

Reply:
Sure, I'll add it as a query param